src/step-10-hooks/index.js

```diff
@@ -0,0 +1,25 @@
+import Fastify from 'fastify'
+
+function buildServer(config) {
+  const opts = {
+    ...config,
+    logger: {
+      level: config.LOG_LEVEL,
+      prettyPrint: config.PRETTY_PRINT,
+    }
+  }
+
+  const fastify = Fastify(opts)
+
+  fastify.register(import('./plugins/authenticate.js'), opts)
+
+  fastify.register(import('./routes/user/index.js'))
+  fastify.register(import('./routes/login.js'))
+  fastify.register(import('./routes/users.js'))
+
+  fastify.log.info('Fastify is ready to go!')
```

Comment:
This is not true. You need to call `.ready()` to have fastify ready to go. You could also do `await fastify` which will do the same.

Reply:
I updated the message to 'fastify is starting up' as this solution didn't mean changing anything (`await` caused knock-on problems in pretty much every other file) and it still serves as a clear example of logging for students.